BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -60,6 +61,8 @@ public final class BlueprintView: UIView {
         
         self.backgroundColor = .white
         addSubview(rootController.view)
+        setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        setContentHuggingPriority(.defaultHigh, for: .vertical)
```

Comment:
I've never seen a view set this on itself. Isn't the CHP something that should come from outside as part of the layout that includes this view (i.e. the storyboard or code that applies constraints to this view)? Why change the defaults?

Reply:
Per the documentation:

> Custom views should set default values for both orientations on creation, based on their content, typically to `UILayoutPriorityDefaultLow` or `UILayoutPriorityDefaultHigh`.

Mostly though, without this, when used in a SwiftUI preview the `.layout(.sizeThatFits)` option gives you a screen-sized preview. And it makes sense to me that a BlueprintView would quite prefer to be the correct size for its contents.